MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -257,15 +265,18 @@ class HomeViewController: UIViewController, FloatingPanelControllerDelegate, Tra
             string: balanceValueString,
             attributes: [
                 NSAttributedString.Key.font: Theme.shared.fonts.homeScreenTotalBalanceValueLabel!,
-                NSAttributedString.Key.foregroundColor: Theme.shared.colors.homeScreenTotalBalanceValueLabel!
+                NSAttributedString.Key.foregroundColor: Theme.shared.colors.homeScreenTotalBalanceValueLabel!,
+                NSAttributedString.Key.kern: -1.43
```

Comment:
What is the reason for the negative kerning?

Reply:
just following the values into Zeplin mock-ups. There are negative values for this label

Reply:
ok no problem then 👍